test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -262,6 +263,28 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #region Join tests
 
+        [Test]
+        [TestModel(@".\elementJoin.rvt")]
+        public void CanSuccessfullyGetJoinedElementsFromElement()
+        {
+            // Arrange - get element from model
+            var element = ElementSelector.ByElementId(184176, true);
+
+            // Act
+            List<Element> joinedElements = element.GetJoinedElements();
+
+            // Assert
+            Assert.AreEqual(3, joinedElements.Count);
+
+            List<int> joinedElementIds = new List<int>();
+            for (int i = 0; i < joinedElements.Count; i++)
+            {
+                joinedElementIds.Add(joinedElements[i].Id);
+            }
+
+            CollectionAssert.AreEqual(new int[] { 207960, 208259, 208422 }, joinedElementIds);
```

Comment:
same, the list of expected values (call it `var expectedElementIds`) should be in `Arrange`